spec/factories/application_drafts.rb

```diff
@@ -14,7 +14,8 @@
 
       after(:create) do |draft|
         draft.students.each do |student|
-          student.update(attributes_for :student, :applicant)
+          student_update_attributes = (attributes_for :student, :applicant).except(:github_handle)
+          student.update(student_update_attributes)
```

Comment:
Great catch!

I wonder if there would be a way to clean this up so as to not have the application draft know about `github_handle`s. What do you think?